apps/assessment_registry/dashboard_schema.py

```diff
@@ -0,0 +1,188 @@
+from dataclasses import dataclass
+import itertools
+import graphene
+from .models import AssessmentRegistry, MethodologyAttribute
+from .enums import (
+    AssessmentRegistryCoordinationTypeEnum,
+    AssessmentRegistryDataCollectionTechniqueTypeEnum,
+)
+from django.db.models import Count
+from django.db import models
+from deep.caches import CacheHelper
+from collections import defaultdict
```

Comment:
```suggestion
import itertools
import graphene

from collections import defaultdict
from dataclasses import dataclass

from django.db.models import Count
from django.db import models

from deep.caches import CacheHelper
from .enums import (
    AssessmentRegistryCoordinationTypeEnum,
    AssessmentRegistryDataCollectionTechniqueTypeEnum,
)
from .models import AssessmentRegistry, MethodologyAttribute
```

Reply:
Lets maintain the order of import.
example:
# future
from __future__ import unicode_literals

# standard library
import json
from itertools import chain

# third-party
import bcrypt

# Django
from django.http import Http404
from django.http.response import (
    Http404,
    HttpResponse,
    HttpResponseNotAllowed,
    StreamingHttpResponse,
    cookie,
)

# local Django
from .models import LogEntry

Reply:
Let's define import in this order
```
python native modules (import datetime)

external modules (import graphene)

internal modules (from project.models import Project)

relative modules (from .models import Lead)
```